src/shared/messaging/test/port-rpc-test.js

```diff
@@ -79,11 +80,12 @@ describe('PortRPC', () => {
     sinon.restore();
   });
 
-  it('should call the method `plusOne` on rpc2', done => {
-    rpc1.call('plusOne', 1, 2, 3, value => {
-      assert.deepEqual(value, [2, 3, 4]);
-      done();
-    });
+  it('should call the method `plusOne` on rpc2', async () => {
+    const { promise, resolve } = promiseWithResolvers();
```

Comment:
This is fine. In _test_ code we can use `Promise.withResolvers` now. In production we still avoid it to support older browsers.